### PR TITLE
Ks successful contribution def

### PR DESCRIPTION
**Pull request by captnseagraves**
Ks successful contribution def

@owocki is this the sort of thing you were expecting?

### CODE REVIEW

File: app/grants/models.py [hunk 1]
```diff
@@ -314,34 +325,12 @@ class ContributionQuerySet(models.QuerySet):
 class Contribution(SuperModel):
     """Define the structure of a subscription agreement."""
 
-    tx_id = models.CharField(max_length=255, default='0x0', help_text=_('The transaction ID of the Contribution.'))
-    from_address = models.CharField(
+    tx_id = models.CharField(
         max_length=255,
         default='0x0',
-        help_text=_('The wallet address tokens are sent from.'),
-    )
```
Comment (owocki) [inline app/grants/models.py]: you just need to `./manage.py makemigrations grants` on this so it creates a migration

File: app/grants/models.py [hunk 2]
```diff
@@ -304,6 +304,17 @@ def __str__(self):
...
+    def successful_contribution(self, kwargs):
+        """Create a contribution object."""
+        contribution_kwargs = {
+            'tx_id': kwargs.tx_id,
+            'gas_price': kwargs.gas_price,
+            'nonce': kwargs.nonce,
+            'subscription': self
+        }
+        contribution = Contribution.objects.create(**contribution_kwargs)
+        # TODO: Email user that there payment has gone through
```
Comment (owocki) [inline app/grants/models.py]: can you generate this email in this PR?

Comment (owocki) [reply app/grants/models.py]: are there any other `TODO`s here?

Comment (captnseagraves) [reply app/grants/models.py]: Sure, I can create the template and wire everything up. Will probably need to polish after the fact.

File: app/grants/models.py (hunk 1 above)
Comment (captnseagraves) [reply app/grants/models.py]: will do

File: app/grants/models.py (hunk 2 above)
Comment (captnseagraves) [reply app/grants/models.py]: I dont think there are any other todo's here

**Pull request merged**